### ISSUE TITLE
SiamFC-R architecture design

### ISSUE THREAD

**Issue by foolwood** [bertinetto/siamese-fc#37]
Hi, Luca.
SiamFC is a great framework for high-speed visual tracking. The default feature extractor is similar to AlexNet. 
To get a competitive preformance to recent state-of-the-art visual tracker, it's helpful to release the ResNet version. 
Since SiamFC is a Fully-Convolutional Network with zero-padding, it's hard to use identity mapping.

I try to use 2 max pooling(3x3) layers in the identity mapping branch to get a compatible resolution, but the performance is degraded a lot.(0.48 on OTB2013)

**Comment by jvlmdr** [comment]
Hey,

Thanks for the `benchmark_results` repository that you maintain!

We use `util/CropMargin.m` rather than pooling to make the sizes compatible at the residual connection when padding is disabled (aka "valid" padding). This is a trivial layer that simply removes a constant number of pixels from all sides.

However, I seem to remember that the ResNet architecture did not give better results than AlexNet after we started using the full ImageNet dataset (in an earlier version of the paper, we excluded some classes in ImagetNet VID).

Hope this helps!
